pynetworktables2js/js/networktables.js

```diff
@@ -546,7 +546,7 @@ const NetworkTables = new function () {
 		globalListeners.add(f);
 		
 		if (immediateNotify === true) {
-			ntCache.forEach(function(k, v){
+			ntCache.forEach(function(v, k){
```

Comment:
This hasn't changed in 5 years. Why does it need to be changed now?

Reply:
Didn't we change `ntCache` to a `Map` this year?

Reply:
Yes, the change to Map is what broke it.